### DISCUSSION TITLE
Next build installing dependencies twice when Deploying with Vercel

### DISCUSSION THREAD

**Discussion by spennybad** [vercel/community#5305] [Help]
### Summary

Hello! I created a website a couple years ago and I am moving over to Node v18 as 16 is getting deprecated soon. I need to now force the installation of the dependencies, and that works fine at first. The override installation command gets run, the build completes, and then for some reason a secondary dependency install runs which does not use the force flag and then the deployment fails.

Any help would be super appreciated!

(please see the attached logs in the steps to reproduce section)

### Steps to Reproduce

[message.txt](https://github.com/vercel/community/files/13780477/message.txt)

**Comment by IrumiDesu7** [comment]
i second this, having the same issue. have you solved this one?

**Comment by leonsilicon** [comment]
Figured it out after a couple of hours of debugging: this happens when you have a top-level folder called `api` in your Next.js project; renaming it to something else fixes the issue!

**Comment by anamhira47** [comment]
Also having this error with a react app anyone know any other workarounds
